### PR TITLE
Fix comment full visibility on mobile

### LINKED ISSUE #1036: Edit option below a comment is not fully visible on mobile
#### Expected behavior

All elements and text in the line below a comment should be fully visible.

screenshot_20171107-070343

#### Actual behavior

The edit word in Czech "Upravit" is partly cut off if the comment is not at the top level of comment conversation.

screenshot_20171107-070331

#### How to reproduce

1. Set language to Czech
2. Find your comment that is not at the top level

* Browser: Brave
* Operating system: Android 7.1.2

### PR DESCRIPTION
**Pull request by sweever**
Fixes #1036

fixing comment full visibility on mobile by adding new css params to CommentFooter .
Changes:

- Changes on ; busy/src/components/CommentFooter/CommentFooter.lss
### Adding new Class ;
```
&__bullet_br:after {
    content: "\2022";
    font-size: 14px;
    margin: 0 8px;
	display: block;
	visibility: hidden;
	line-height: 0.5;
	
	@media only screen and (min-width: 468px){
		display: unset;
		visibility: unset;
		line-height: 0.5;
    }
  }
```
- Changes on ; busy/src/components/CommentFooter/Buttons.js
### implement the new class ;
```
line 225 : <span className="CommentFooter__bullet_br" />
```

### CODE REVIEW

**Comment by Sekhmet** [comment]
image
In my case (1600x900 resolution) it breaks footer into two lines, but it shouldn't.
I'm not sure how do we want it to look like though.
cc @bonustrack

**Comment by sweever** [comment]
i suggest to round it betwin @media mobile screen .

File: src/components/CommentFooter/CommentFooter.less [hunk 1]
```diff
@@ -4,6 +4,12 @@
   padding: 8px 0;
   color: @blue-botticelli;
   line-height: 14px;
+  
+  span:nth-child(7) > span{
```
Comment (jm90m) [inline src/components/CommentFooter/CommentFooter.less]: idk if this is the right approach.. select the 7th child child then the child span of that.... @Sekhmet what are your thoughts?

Comment (Sekhmet) [reply src/components/CommentFooter/CommentFooter.less]: Yeah, we should have a class for this.

**Comment by jm90m** [comment]
need a deploy on this @bonustrack

File: src/components/CommentFooter/CommentFooter.less [hunk 2]
```diff
@@ -16,6 +16,21 @@
...
 
+  &__bullet_br:after {
+    content: "\2022";
+    font-size: 14px;
+    margin: 0 8px;
+	  display: block;
+	  visibility: hidden;
+  	line-height: 0.5;
+	
+	  @media only screen and (min-width: 468px){
```
Comment (jm90m) [inline src/components/CommentFooter/CommentFooter.less]: Should probably used a predefined media query in `busy/src/styles/modules/variables.less`
or create one

**Comment by bonustrack** [comment]
Can you change the base of this PR to be `master` instead of `new-design`?

**Comment by bonustrack** [comment]
Can you resolve conflict @sweever ?

File: src/components/CommentFooter/CommentFooter.less [hunk 3]
```diff
@@ -16,6 +16,21 @@
...
 
+  &__bullet_br:after {
+    content: "\2022";
+    font-size: 14px;
+    margin: 0 8px;
+	  display: block;
+	  visibility: hidden;
+  	line-height: 0.5;
+	
+	  @media @very-small {
```
Comment (bonustrack) [inline src/components/CommentFooter/CommentFooter.less]: This line is breaking the deployment of the app with this issue:
```
oader/lib/index.js??postcss!node_modules/less-loader/dist/cjs.js!src/components/CommentFooter/CommentFooter.less:
       1 module
       
       ERROR in ./node_modules/css-loader?{"importLoaders":1}!./node_modules/postcss-loader/lib?{"ident":"postcss","plugins":[null]}!./node_modules/less-loader/dist/cjs.js!./src/components/CommentFooter/CommentFooter.less
       Module build failed:
       
       
       @media @very-small {
       ^
       Variable @very-small is undefined
       in /tmp/build_f410f0737e336c27aa8dc7e1c220ce61/busyorg-busy-bb35cb9/src/components/CommentFooter/CommentFooter.less (line 27, column 10)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! busy@2.0.0 build:client: `webpack -p --config ./webpack/webpack.prod.config.js`
npm ERR! Exit status 2
```

Comment (sweever) [reply src/components/CommentFooter/CommentFooter.less]: the variable file is not imported i will fix it .

**Pull request closed without merging**